File: dal-dao-gen/dal_ftp/63/java/Entity/FltOrdersTmp.java
package com.ctrip.dal.test.test4;

import java.math.BigDecimal;
import java.sql.Timestamp;

import com.ctrip.platform.dal.dao.DalPojo;

public class FltOrdersTmp implements DalPojo {
	private Integer recordId;
	private Integer orderId;
	private String passengerName;
	private Integer sequence;
	private Integer accCheckId;
	private BigDecimal price;
	private BigDecimal tax;
	private BigDecimal oilFee;
	private BigDecimal sendticketfee;
	private BigDecimal insurancefee;
	private BigDecimal serviceFee;
	private BigDecimal refund;
	private BigDecimal delAdjustAmount;
	private BigDecimal adjustedAmount;
	private String orderStatus;
	private String remark;
	private Timestamp createTime;
	private Timestamp confirmTime;
	private String dailyConfirmFlag;
	private Integer dealID;
	private BigDecimal cost;
	private Timestamp datachangeLasttime;
	public Integer getRecordId() {
		return recordId;
	}

	public void setRecordId(Integer recordId) {
		this.recordId = recordId;
	}

	public Integer getOrderId() {
		return orderId;
	}

	public void setOrderId(Integer orderId) {
		this.orderId = orderId;
	}

	public String getPassengerName() {
		return passengerName;
	}

	public void setPassengerName(String passengerName) {
		this.passengerName = passengerName;
	}

	public Integer getSequence() {
		return sequence;
	}

	public void setSequence(Integer sequence) {
		this.sequence = sequence;
	}

	public Integer getAccCheckId() {
		return accCheckId;
	}

	public void setAccCheckId(Integer accCheckId) {
		this.accCheckId = accCheckId;
	}

	public BigDecimal getPrice() {
		return price;
	}

	public void setPrice(BigDecimal price) {
		this.price = price;
	}

	public BigDecimal getTax() {
		return tax;
	}

	public void setTax(BigDecimal tax) {
		this.tax = tax;
	}

	public BigDecimal getOilFee() {
		return oilFee;
	}

	public void setOilFee(BigDecimal oilFee) {
		this.oilFee = oilFee;
	}

	public BigDecimal getSendticketfee() {
		return sendticketfee;
	}

	public void setSendticketfee(BigDecimal sendticketfee) {
		this.sendticketfee = sendticketfee;
	}

	public BigDecimal getInsurancefee() {
		return insurancefee;
	}

	public void setInsurancefee(BigDecimal insurancefee) {
		this.insurancefee = insurancefee;
	}

	public BigDecimal getServiceFee() {
		return serviceFee;
	}

	public void setServiceFee(BigDecimal serviceFee) {
		this.serviceFee = serviceFee;
	}

	public BigDecimal getRefund() {
		return refund;
	}

	public void setRefund(BigDecimal refund) {
		this.refund = refund;
	}

	public BigDecimal getDelAdjustAmount() {
		return delAdjustAmount;
	}

	public void setDelAdjustAmount(BigDecimal delAdjustAmount) {
		this.delAdjustAmount = delAdjustAmount;
	}

	public BigDecimal getAdjustedAmount() {
		return adjustedAmount;
	}

	public void setAdjustedAmount(BigDecimal adjustedAmount) {
		this.adjustedAmount = adjustedAmount;
	}

	public String getOrderStatus() {
		return orderStatus;
	}

	public void setOrderStatus(String orderStatus) {
		this.orderStatus = orderStatus;
	}

	public String getRemark() {
		return remark;
	}

	public void setRemark(String remark) {
		this.remark = remark;
	}

	public Timestamp getCreateTime() {
		return createTime;
	}

	public void setCreateTime(Timestamp createTime) {
		this.createTime = createTime;
	}

	public Timestamp getConfirmTime() {
		return confirmTime;
	}

	public void setConfirmTime(Timestamp confirmTime) {
		this.confirmTime = confirmTime;
	}

	public String getDailyConfirmFlag() {
		return dailyConfirmFlag;
	}

	public void setDailyConfirmFlag(String dailyConfirmFlag) {
		this.dailyConfirmFlag = dailyConfirmFlag;
	}

	public Integer getDealID() {
		return dealID;
	}

	public void setDealID(Integer dealID) {
		this.dealID = dealID;
	}

	public BigDecimal getCost() {
		return cost;
	}

	public void setCost(BigDecimal cost) {
		this.cost = cost;
	}

	public Timestamp getDatachangeLasttime() {
		return datachangeLasttime;
	}

	public void setDatachangeLasttime(Timestamp datachangeLasttime) {
		this.datachangeLasttime = datachangeLasttime;
	}

}

File: dal-dao-gen/dal_ftp/63/cs/Entity/FltOrdersTmp.cs
using System;
using Arch.Data.Orm;

namespace com.ctrip.dal.test.test4.Entity.DataModel
{
    /// <summary>
    /// _FltOrdersTmp
    /// </summary>
    [Serializable]
    [Table(Name = "_FltOrdersTmp")]
    public partial class FltOrdersTmp
    {
        /// <summary>
        /// 获取或设置RecordId
        /// </summary>
        [Column(Name = "RecordId",Length=10),ID,PK]
        public int RecordId { get; set; }
        /// <summary>
        /// 获取或设置OrderId
        /// </summary>
        [Column(Name = "OrderId",Length=10)]
        public int OrderId { get; set; }
        /// <summary>
        /// 获取或设置PassengerName
        /// </summary>
        [Column(Name = "PassengerName",Length=40)]
        public string PassengerName { get; set; }
        /// <summary>
        /// 获取或设置Sequence
        /// </summary>
        [Column(Name = "Sequence",Length=5)]
        public short Sequence { get; set; }
        /// <summary>
        /// 获取或设置AccCheckId
        /// </summary>
        [Column(Name = "AccCheckId",Length=10)]
        public int AccCheckId { get; set; }
        /// <summary>
        /// 获取或设置Price
        /// </summary>
        [Column(Name = "Price",Length=19)]
        public decimal Price { get; set; }
        /// <summary>
        /// 获取或设置Tax
        /// </summary>
        [Column(Name = "Tax",Length=19)]
        public decimal Tax { get; set; }
        /// <summary>
        /// 获取或设置OilFee
        /// </summary>
        [Column(Name = "OilFee",Length=19)]
        public decimal OilFee { get; set; }
        /// <summary>
        /// 获取或设置Sendticketfee
        /// </summary>
        [Column(Name = "Sendticketfee",Length=19)]
        public decimal Sendticketfee { get; set; }
        /// <summary>
        /// 获取或设置Insurancefee
        /// </summary>
        [Column(Name = "Insurancefee",Length=19)]
        public decimal Insurancefee { get; set; }
        /// <summary>
        /// 获取或设置ServiceFee
        /// </summary>
        [Column(Name = "ServiceFee",Length=19)]
        public decimal ServiceFee { get; set; }
        /// <summary>
        /// 获取或设置Refund
        /// </summary>
        [Column(Name = "Refund",Length=19)]
        public decimal Refund { get; set; }
        /// <summary>
        /// 获取或设置delAdjustAmount
        /// </summary>
        [Column(Name = "delAdjustAmount",Length=19)]
        public decimal DelAdjustAmount { get; set; }
        /// <summary>
        /// 获取或设置AdjustedAmount
        /// </summary>
        [Column(Name = "AdjustedAmount",Length=19)]
        public decimal AdjustedAmount { get; set; }
        /// <summary>
        /// 获取或设置OrderStatus
        /// </summary>
        [Column(Name = "OrderStatus",Length=1)]
        public char OrderStatus { get; set; }
        /// <summary>
        /// 获取或设置Remark
        /// </summary>
        [Column(Name = "Remark",Length=200)]
        public string Remark { get; set; }
        /// <summary>
        /// 获取或设置CreateTime
        /// </summary>
        [Column(Name = "CreateTime",Length=23)]
        public DateTime CreateTime { get; set; }
        /// <summary>
        /// 获取或设置ConfirmTime
        /// </summary>
        [Column(Name = "ConfirmTime",Length=23)]
        public DateTime? ConfirmTime { get; set; }
        /// <summary>
        /// 获取或设置DailyConfirmFlag
        /// </summary>
        [Column(Name = "DailyConfirmFlag",Length=1)]
        public char? DailyConfirmFlag { get; set; }
        /// <summary>
        /// 获取或设置DealID
        /// </summary>
        [Column(Name = "DealID",Length=10)]
        public int? DealID { get; set; }
        /// <summary>
        /// 获取或设置Cost
        /// </summary>
        [Column(Name = "Cost",Length=19)]
        public decimal? Cost { get; set; }
        /// <summary>
        /// 获取或设置DataChangeLastTime
        /// </summary>
        [Column(Name = "DataChangeLastTime",Length=23)]
        public DateTime? DataChangeLastTime { get; set; }
    }
}